pkgs/by-name/to/touche/package.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchNpmDeps,
+
+  meson,
+  ninja,
+  pkg-config,
+  nodejs,
+  npmHooks,
+  desktop-file-utils,
+  appstream-glib,
+  gjs,
+  gobject-introspection,
+  glib,
+  gtk3,
+  python3,
+  wrapGAppsHook4,
+
+  libadwaita,
+  xorg,
+}:
+let
+  fqpn = "com.github.joseexposito.touche";
```

Comment:
Why isn't the binary's name just `touche`? This makes it quite annoying to launch, especially since it has no `desktopItem` (which you should also consider adding, if possible).

Reply:
I have no idea, and upstream is apparently this way by default. A symlink won't hurt though

Reply:
It seems that the AUR package exports it as `touche`: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=touche#n56. That's good enough for me